pkg/security/config/config.go

```diff
@@ -119,6 +123,8 @@ func NewConfig(cfg *config.Config) (*Config, error) {
 		MapDentryResolutionEnabled:         aconfig.Datadog.GetBool("runtime_security_config.map_dentry_resolution_enabled"),
 		RemoteTaggerEnabled:                aconfig.Datadog.GetBool("runtime_security_config.remote_tagger"),
 		LogPatterns:                        aconfig.Datadog.GetStringSlice("runtime_security_config.log_patterns"),
+		SelfTestEnabled:                    true,
```

Comment:
I think we should allow to disable it. I would enable the self test by default and disable the "test at start" by default. WDYT ?